examples/hello_world.cz

```diff
@@ -0,0 +1,3 @@
+fun hello() {
```

Comment:
Sorry to keep requesting changes, but you never call `hello` anywhere so this program doesn't do anything.

Reply:
yup, got it